v22.2/upgrade-cockroach-version.md

```diff
@@ -115,7 +115,7 @@ By default, after all nodes are running the new version, the upgrade process wil
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
-    > SET CLUSTER SETTING cluster.preserve_downgrade_option = '21.2';
+    > SET CLUSTER SETTING cluster.preserve_downgrade_option = '22.1';
```

Comment:
Did you want to use the same liquid variable that you added elsewhere, or does that not work here for some reason?